README.md

```diff
@@ -1,3 +1,108 @@
-# Spark Fat Jar Template
+# Setting up a Scheduled Spark Scala job on AWS
 
-Read me will contain article in future. 
\ No newline at end of file
+Have you written a Scala Spark job that processes a massive amount of data on a stupid amount of RAM and you want to figure out a way to run it daily/weekly/monthly on a schedule on [AWS](https://aws.amazon.com)?
+Well, I did and I could not find a good tutorial on the full process to get the spark job running, so here is mine.
+Included in this article and accompanying repository should be everything you need to take your Scala Spark job and get it running on AWS [Data Pipeline](https://aws.amazon.com/datapipeline/) and [EMR](https://aws.amazon.com/emr/).
+
+## Code Repo
+
+This tutorial is not going to walk you through the process of actually writing your specific Scala Spark job to do whatever number crunching you need. 
+There are [plenty of](https://www.analyticsvidhya.com/blog/2017/01/scala/) [resources](https://spark.apache.org/docs/latest/quick-start.html) [available](https://www.coursera.org/learn/scala-spark-big-data) already to get you started on that. 
+The code template for setting up a Spark Scala job is available in [this GitHub repo](https://github.com/jessebrizzi/spark-fat-jar-template).
+
+Assuming that you have already written your Spark Job locally in something like [IntelliJ](https://www.jetbrains.com/idea/) or in the cloud with something like [Zeppelin](https://zeppelin.apache.org/) and [AWS EMR](https://aws.amazon.com/emr/) and you are only using the [AWS Java SDK](https://aws.amazon.com/sdk-for-java/) to connect to your AWS data stores, you should be able to drop your code in the [Main](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/src/main/scala/SparkJob.scala#L4) function of [SparkJob.scala](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/src/main/scala/SparkJob.scala) and and run the [deploy.sh](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/deploy.sh) script to upload the fat jar to your S3 bucket. 
```

Comment:
Optional:
Assuming that you have already written your Spark Job and you are only using the [AWS Java SDK](https://aws.amazon.com/sdk-for-java/) to connect to your AWS data stores

Reply:
`and and ` -> `and`